environment.yml

```diff
@@ -56,5 +56,6 @@ dependencies:
   - pytest-rerunfailures
   - pytest-timeout
   - pytest-xdist
+  - pytest-xvfb
```

Comment:
What does this do on mac/windows?

Reply:
Nothing. No harm in having it installed (except for storage space). And if xvfb on Linux is not installed, it won't complain either.